nifi/nifi-nar-bundles/nifi-kite-bundle/nifi-kite-processors/pom.xml

```diff
@@ -0,0 +1,160 @@
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+  -->
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.apache.nifi</groupId>
+    <artifactId>nifi-kite-bundle</artifactId>
+    <version>0.0.1-incubating-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>nifi-kite-processors</artifactId>
+  <packaging>jar</packaging>
+  <name>Kite Hadoop Processors</name>
+
+  <properties>
+    <kite.version>0.18.0</kite.version>
+    <guava.version>11.0.2</guava.version>
+    <jsr305.version>2.0.1</jsr305.version>
+    <junit.version>4.10</junit.version>
+  </properties>
+
+  <dependencies>
+    <!-- NiFi -->
+
+    <dependency>
+      <groupId>org.apache.nifi</groupId>
+      <artifactId>nifi-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.nifi</groupId>
+      <artifactId>nifi-utils</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.nifi</groupId>
+      <artifactId>nifi-processor-utils</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.nifi</groupId>
+      <artifactId>nifi-flowfile-packager</artifactId>
+    </dependency>
+
+    <!-- Kite -->
+
+    <dependency>
+      <groupId>org.kitesdk</groupId>
+      <artifactId>kite-data-core</artifactId>
+      <version>${kite.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.kitesdk</groupId>
+      <artifactId>kite-data-hive</artifactId>
+      <version>${kite.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.kitesdk</groupId>
+      <artifactId>kite-data-hbase</artifactId>
+      <version>${kite.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.kitesdk</groupId>
+      <artifactId>kite-hadoop-dependencies</artifactId>
+      <type>pom</type>
+      <version>${kite.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.kitesdk</groupId>
+      <artifactId>kite-hbase-dependencies</artifactId>
+      <type>pom</type>
+      <scope>provided</scope>
+      <optional>true</optional>
+      <version>${kite.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>${guava.version}</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <!-- avoid warnings by bundling annotations -->
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>jsr305</artifactId>
+      <scope>compile</scope>
+      <version>${jsr305.version}</version>
+    </dependency>
+
+    <dependency>
+      <!-- avoid warnings by bundling annotations -->
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>annotations</artifactId>
```

Comment:
This is an LGPL dependency, so you can't include it. ([ref asf license guidelines](http://www.apache.org/legal/resolved.html#category-x))

There is a cleanroom reimplementation that is appropriately licensed: https://github.com/stephenc/findbugs-annotations